distributed/client.py

```diff
@@ -807,6 +820,120 @@ class VersionsDict(TypedDict):
     client: dict[str, dict[str, Any]]
 
 
+_T_LowLevelGraph: TypeAlias = dict[Key, tuple]
+
+
+def is_nested(iterable):
```

Comment:
Let's prefix this with `_` to mark this as private
```suggestion
def _is_nested(iterable):
```